packages/terra-consumer-nav/src/components/nav-logo/NavLogo.jsx

```diff
@@ -41,9 +42,15 @@ const NavLogo = ({
   link,
   ...customProps
 }) => {
+  const imgProps = {
+    className: cx('img'),
+  };
+  if (url) {
+    imgProps.src = url;
+  }
   const image = (link) ?
-    <SmartLink {...link}> <img className={cx('img')} src={url} alt={altText} /> </SmartLink> :
-    <img className={cx('img')} src={url} alt={altText} />;
+    <SmartLink {...link}> <img alt={altText} {...imgProps} /> </SmartLink> :
+    <img alt={altText} {...imgProps} />;
```

Comment:
np: perhaps split this out into a var instead of having it written out twice

Reply:
make sense [updated](https://github.com/cerner/terra-consumer/pull/60/files)